admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/server/service/persistence/PersistenceManagerImpl.java

```diff
@@ -421,8 +421,8 @@ protected void postFetchValidation(PersistencePackage persistencePackage, Persis
                         .filter(sectionCrumb -> sectionCrumb.getOriginalSectionIdentifier() != null)
                         .filter(sectionCrumb -> sectionCrumb.getOriginalSectionIdentifier().equals(foreignKey.getManyToField()))
                         .findFirst();
-                    if (property != null && sectionCrumbOptional.isPresent()
-                        && !property.getValue().equals(sectionCrumbOptional.get().getSectionId())) {
+                    property.setValue(sectionCrumbOptional.get().getSectionId());
```

Comment:
You can't change the property.  The solution should be similar to https://github.com/BroadleafCommerce/Enterprise/pull/1452 in that we need to take the property id and find its parent.  The issue here is that this code is in framework and the logic we need is in Enterprise.  So we will likely need an ExtensionHandler to call into Enterprise to get the parent id.